src/core/src/preprocess/pre_post_process.cpp

```diff
@@ -249,6 +249,11 @@ InputTensorInfo& InputTensorInfo::set_shape(const PartialShape& shape) {
     return *this;
 }
 
+InputTensorInfo& InputTensorInfo::set_from(const ov::Tensor& runtime_tensor) {
```

Comment:
Can we add the same method for OutputTensorInfo?

Reply:
For output we can reuse only 'element_type', there is no 'set_shape' for OutputTensorInfo. This is because 'postprocessing' don't support operations like 'resize'. So there seems no need to wrap one line of code (which is equivalent to `ppp.output().set_element_type(output_tensor.get_element_type())`